README.md

```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual Studio 2015](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx)
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python](https://www.python.org/downloads/) (add to PATH)
```

Comment:
@Zangetsu38 And that's why we specify 2.7+ aka 2.7 or higher - up to the newest version. If we don't specify that, one could even assume that 2.6 or something would work, because the **minimum** required version is not specified. While with the "or higher", one could assume anything above 2.7 will work, including 3.* versions. I don't see how it would confuse anyone, but rather make them aware to update to a newer version (the newest 2.* or 3.*), if they're on a version, that's older.